src/Console/InstallCommand.php

```diff
@@ -31,10 +31,10 @@ public function handle()
         // NPM Packages...
         $this->updateNodePackages(function ($packages) {
             return [
-                '@tailwindcss/ui' => '^0.7.2',
+                '@tailwindcss/forms' => '^0.2.1',
                 'alpinejs' => '^2.7.3',
                 'postcss-import' => '^12.0.1',
-                'tailwindcss' => '^1.8.0',
+                'tailwindcss' => 'npm:@tailwindcss/postcss7-compat@^2.0.1',
```

Comment:
I think you should add here also this line:

```suggestion
                'tailwindcss' => 'npm:@tailwindcss/postcss7-compat@^2.0.1',
                'autoprefixer' => '^9.8.6',
```